quick-start/README.md

```diff
@@ -1 +1,14 @@
 # Quick Start
+
+This is a quick example of how to retrieve your first secret from HashiCorp [Vault](https://www.vaultproject.io/), using HashiCorp's official [Go client library](https://pkg.go.dev/github.com/hashicorp/vault/api).
+
+## How To Use
+
+1. Start the Vault dev server with `./run.sh`.
+2. Run the sample code with `go run main.go`.
+
+You just wrote and read your first Vault secret!
+
+## Next Steps
+
+For a more realistic example of a runnable web app connecting to Vault using the AppRole authentication method, check out the code in `../sample-app`.
```

Comment:
I think it would make it better to make "sample app" a link to the sample app directory.